<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphSettingsDialogB</class>
 <widget class="QDialog" name="GraphSettingsDialogB">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>152</width>
    <height>174</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="selectBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>134</width>
     <height>114</height>
    </rect>
   </property>
   <property name="title">
    <string>Select Axis Scaling</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QCheckBox" name="autoscaleYcheckbox">
      <property name="text">
       <string>Autoscale Y</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="autoscaleXcheckbox">
      <property name="text">
       <string>Autoscale X</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
